src/valgrind.supp

```diff
@@ -0,0 +1,28 @@
+{
+   <registered functions>
+   Memcheck:Leak
+   ...
+   fun:AR_RegisterFuncs
+   fun:RedisModule_OnLoad
+   fun:moduleLoad
+   fun:moduleLoadFromQueue
+   fun:main
+}
```

Comment:
The registered functions we build are instantiated on module load and should always be accessible. They are not freed when the server is shut down or if the module is unloaded, which I think is trivial.